scripts/globals/mobskills/murk.lua

```diff
@@ -6,7 +6,6 @@
 -- Utsusemi/Blink absorb: Ignores shadows
 -----------------------------------
 require("scripts/globals/monstertpmoves")
-require("scripts/globals/settings")
 require("scripts/globals/status")
 require("scripts/globals/msg")
```

Comment:
msg doesn't appear to be requred here (none of mine from release has this).  Also I clicked on the plus under the right side, not sure why it moved to the left.

Reply:
msg is used on line 34 for `skill:setMsg(tpz.msg.basic.SKILL_MISS)`